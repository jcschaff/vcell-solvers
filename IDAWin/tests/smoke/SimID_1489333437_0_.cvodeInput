SOLVER CVODE
STARTING_TIME 0.0
ENDING_TIME 1.0
RELATIVE_TOLERANCE 1.0E-9
ABSOLUTE_TOLERANCE 1.0E-9
MAX_TIME_STEP 1.0
OUTPUT_TIMES 8
0.0 1.0E-5 1.0E-4 0.001 0.01 0.1 0.5 1.0
NUM_EQUATIONS 2
ODE s2 INIT 4.0;
	 RATE  - ((2.0 * (-1.0 + (0.5 * s2)) * s2) - (4.0 * (3.5 - (0.5 * s2))));
ODE s4 INIT 0.0;
	 RATE 0.0;

