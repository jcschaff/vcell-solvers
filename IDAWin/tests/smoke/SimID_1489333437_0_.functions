##---------------------------------------------
##  /Users/jimschaff/.vcell/simdata/temp/SimID_1489333437_0_.functions
##---------------------------------------------

Compartment::J_r0; ((2.0 * (-1.0 + (0.5 * s2)) * s2) - (4.0 * (3.5 - (0.5 * s2)))); ; Nonspatial_VariableType; false
Compartment::J_r1; 0.0; ; Nonspatial_VariableType; false
Compartment::s0; (2.0 * (-1.0 + (0.5 * s2))); ; Nonspatial_VariableType; false
Compartment::s1; (2.0 * (3.5 - (0.5 * s2))); ; Nonspatial_VariableType; false
Compartment::s3;  - s4; ; Nonspatial_VariableType; false

