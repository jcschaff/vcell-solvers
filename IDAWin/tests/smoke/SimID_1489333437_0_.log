IDAData logfile
IDAData text format version 1
SimID_1489333437_0_.ida
