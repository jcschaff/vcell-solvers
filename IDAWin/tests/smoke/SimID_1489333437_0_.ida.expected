t:s2:s4:
0.00000000000000000E+00	4.00000000000000000E+00	0.00000000000000000E+00
1.00000000000000008E-05	3.99998000095349759E+00	0.00000000000000000E+00
1.00000000000000005E-04	3.99980008070643711E+00	0.00000000000000000E+00
1.00000000000000002E-03	3.99800797408532871E+00	0.00000000000000000E+00
1.00000000000000002E-02	3.98077784306128457E+00	0.00000000000000000E+00
1.00000000000000006E-01	3.86170998966032331E+00	0.00000000000000000E+00
5.00000000000000000E-01	3.74758417187014503E+00	0.00000000000000000E+00
1.00000000000000000E+00	3.74179783773998942E+00	0.00000000000000000E+00
